content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Using InfinyOn Cloud Backend
+menu: InfinyOn Cloud Tutorial
+weight: 60
+---
+
+This tutorial will walk you through setting up an InfinyOn account, and using
+Fluvio, both as a standalone command, and as part of a larger program.
+
+## Basic Setup
+
+There are 3 main steps for setting up for the Cloud: Installing the CLI,
+registering for a Cloud account, and finally linking the two together.
+
```

Comment:
![installation-steps](https://user-images.githubusercontent.com/26554085/188802682-c85b1d75-8b38-49f9-8ed4-63489dffb8ce.svg)

Reply:
Visuals sometimes makes the content easier to digest.